File: packages/expo/ios/ClerkUserProfileViewManager.m
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(ClerkUserProfileViewManager, RCTViewManager)

RCT_EXPORT_VIEW_PROPERTY(isDismissible, NSNumber)
RCT_EXPORT_VIEW_PROPERTY(onProfileEvent, RCTBubblingEventBlock)

@end

File: packages/expo/ios/ClerkUserProfileViewManager.swift
import React

@objc(ClerkUserProfileViewManager)
class ClerkUserProfileViewManager: RCTViewManager {

  override static func requiresMainQueueSetup() -> Bool {
    return true
  }

  override func view() -> UIView! {
    return ClerkUserProfileNativeView()
  }
}
